pkg/services/ngalert/state/state.go

```diff
@@ -162,6 +163,9 @@ func (a *State) resultNoData(alertRule *models.AlertRule, result eval.Result) {
 }
 
 func (a *State) NeedsSending(resendDelay time.Duration) bool {
+	if a.Stale { // stale state is deleted from cache and will not be sent many times
```

Comment:
I thought we want to send resolved (stale) states to the Alertmanager so the resolved notification has the correct values?